CMakeLists.txt

```diff
@@ -12,11 +12,11 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-# We require at least CMake V3.3.2
-cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
+# We require at least CMake V3.6.3
+cmake_minimum_required(VERSION 3.6.3 FATAL_ERROR)
```

Comment:
Remember to update this to whichever version we require in the end. Most likely the check below can go as well if it's 3.10.

Reply:
Not sure I understood, I actually update it to 3.6.3 cause it works with 3.6.3, why imposing 3.10 ?

Reply:
Will do it for the modules

Reply:
Even better! I thought we required a newer one.